pkg/vulnerability/vulnerability.go

```diff
@@ -58,6 +74,12 @@ func (c Client) FillInfo(vulns []types.DetectedVulnerability, light bool) {
 	}
 }
 
+func (c Client) getVendorSeverity(vuln *types.DetectedVulnerability, reportType string) {
+	if vs, ok := vuln.Vulnerability.VendorSeverity[reportType]; ok {
```

Comment:
If `ok` is false, we should display the severity of NVD. I couldn't find such a process. Is there something to do it?

Reply:
By default, `GetVulnerability` includes a vulnerability struct that already has a `Severity` populated in it. 

You can see it here https://github.com/aquasecurity/trivy/blob/a4209368fdf3ebb76d9b99efd295993ad224eb71/pkg/vulnerability/vulnerability.go#L54

But there's another point that I think is possible, I added some tests to show that: https://github.com/aquasecurity/trivy/pull/433/commits/6037403fc4429c516bab404d5d3ba6eec6854bf3 and https://github.com/aquasecurity/trivy/pull/433/commits/9b54a4f904d5298cccd1d0940e60b973d421ba71

Good catch!

Reply:
`GetVulnerability` populates a severity which is sometimes not from NVD. 

nit: BTW, I feel it is weird `getXXX` doesn't return anything.